plugins/modules/apk.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+# shellcheck shell=ash
+# Copyright (c) 2024 Krzysztof Bialek/Markus Weippert
+# GNU General Public License v3.0 (see https://www.gnu.org/licenses/gpl-3.0.txt)
+
+PARAMS="
+    name=pkg/str/r
+    state/str//present
+    update_cache/bool
+    no_cache/bool
+    force_broken_world/bool
+"
+
+query_package() {
+    # apk info returns 0 if installed, 1 if not
+    apk info -e "$1" >/dev/null 2>&1
+}
+
+install_packages() {
+    local _IFS pkg pkgs_to_install
+    _IFS="$IFS"; IFS=","; set -- $name; IFS="$_IFS"
+    
+    for pkg; do
+        if ! query_package "$pkg"; then
+            pkgs_to_install="$pkgs_to_install $pkg"
+        fi
+    done
+
+    [ -n "$pkgs_to_install" ] || return 0
+
+    [ -n "$_ansible_check_mode" ] || {
+        # shellcheck disable=SC2086
+        try apk add $update_cache $no_cache $force_broken_world $pkgs_to_install
```

Comment:
Probably only `pkgs_to_install` needs the SC disable. In that case, we should apply the good practice to the other params.
```suggestion
        try apk add "$update_cache" "$no_cache" "$force_broken_world" $pkgs_to_install
```